Bottom: d89d2f10eb6998af6a91eff80f6a173702af6ef7
Top:    9fb7e31076ee1cc616290ce1f02731c95c98d60b
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-12 12:08:00 +0800

Refresh of tuntap-introduce-tx-skb-ring

---

diff --git a/include/linux/net.h b/include/linux/net.h
index 29b5dbc..3909d8e 100644
--- a/include/linux/net.h
+++ b/include/linux/net.h
@@ -132,7 +132,7 @@ struct module;
 struct proto_ops {
 	int		family;
 	struct module	*owner;
-	int		(*peek_len)  (struct socket *sock);
+	int		(*peek_len)	(struct socket *sock);
 	int		(*release)   (struct socket *sock);
 	int		(*bind)	     (struct socket *sock,
 				      struct sockaddr *myaddr,
